.github/scripts/print_latest_commits.py

```diff
@@ -56,7 +58,21 @@ def get_commit_results(commit: str, results: Dict[str, Any]) -> List[Dict[str, A
             )._asdict())
     return workflow_checks
 
-def isGreen(results: List[Dict[str, Any]]) -> bool:
+def isGreen(commit: str, results: Dict[str, Any]) -> bool:
+    workflow_checks = get_commit_results(commit, results)
+
+    for check in workflow_checks:
+        regex = "^linux-binary+|^android-tests+|^windows-binary+"
+        if check['workflowName'] in ["pull", "trunk", "lint"] and check['conclusion'] != 'success':
+            if check['name'] == "pull / win-vs2019-cuda11.3-py3" and check['conclusion'] == 'skipped':
+                pass
```

Comment:
if we're using check['workflowName'] alot, it might be better to make them variables. 

also if there's duplicated tests, can we remove them from trunk then?